tracer/src/Datadog.Trace/ClrProfiler/NativeCallTargetDefinition.cs

```diff
@@ -8,6 +8,30 @@
 
 namespace Datadog.Trace.ClrProfiler
 {
+ // !                                         ██
+ //                                         ██░░██
+ //                                       ██░░░░░░██
+ //                                     ██░░░░░░░░░░██
+ //                                     ██░░░░░░░░░░██
+ //                                   ██░░░░░░░░░░░░░░██
+ //                                 ██░░░░░░██████░░░░░░██
+ //                                 ██░░░░░░██████░░░░░░██
+ //                               ██░░░░░░░░██████░░░░░░░░██
+ //                               ██░░░░░░░░██████░░░░░░░░██
+ //                             ██░░░░░░░░░░██████░░░░░░░░░░██
+ //                           ██░░░░░░░░░░░░██████░░░░░░░░░░░░██
+ //                           ██░░░░░░░░░░░░██████░░░░░░░░░░░░██
+ //                         ██░░░░░░░░░░░░░░██████░░░░░░░░░░░░░░██
+ //                         ██░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░██
+ //                       ██░░░░░░░░░░░░░░░░██████░░░░░░░░░░░░░░░░██
+ //                       ██░░░░░░░░░░░░░░░░██████░░░░░░░░░░░░░░░░██
+ //                     ██░░░░░░░░░░░░░░░░░░██████░░░░░░░░░░░░░░░░░░██
+ //                     ██░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░██
+ //                       ██████████████████████████████████████████
+ //
+ // If you happen to change the layout of this structure,
+ // this will lead to an AccessViolationException in netCore when using a more recent version of the nuget.
```

Comment:
@tonyredondo I think we concluded that any changes would need to go into a new struct? If so, we could add that to the end of the message: `If you need to modify the definition, create a new interface NativeCallTargetDefinition# that will be consumed by the native layer`

Reply:
That’s correct, the only changes allowed here is modifying field names.

Reply:
done in newer commit